worldeditadditions/utils/parse/init.lua

```diff
@@ -5,11 +5,16 @@
 -- ██      ██   ██ ██   ██ ███████ ███████
 
 local axes = dofile(worldeditadditions.modpath.."/utils/parse/axes.lua")
+local parse = dofile(worldeditadditions.modpath.."/utils/parse/axes.lua")
+local key_instance = dofile(worldeditadditions.modpath.."/utils/parse/key_instance.lua")
 
-worldeditadditions.parse = {
-	axes = axes.parse_axes,
-	axis_name = axes.parse_axis_name
-}
+worldeditadditions.key_instance = key_instance
+worldeditadditions.parse = parse
```

Comment:
Not sure I'm a fan of setting `worldeditadditions.parse` to the object returned from importing the axis parser. This does not make it clear which function to call to parse axes, and means you have to navigate to another file to find what you want?

Also, I intend to refactor the other parsers in this directory here too to return a function that then gets set like I did previously here, as in

```lua
worldeditadditions.parse = {
	axes = axes.parse_axes,
	axis_name = axes.parse_axis_name
}
```

If we set `worldeditadditions.parse` equal to the return value of `dofile`ing your new parser, this would make it difficult / confusing to attach those other parsers.

Your parser has a number of helper functions and that's fine, but they don't *all* want to be exposed here.

Reply:
> This does not make it clear which function to call to parse axes, and means you have to navigate to another file to find what you want?

I could add a comment with that information. I'd rather not set it piece by piece as that's a lot of work and if I add any more functions I'd have to manually add them here too.

>If we set worldeditadditions.parse equal to the return value of dofileing your new parser, this would make it difficult / confusing to attach those other parsers.
>Your parser has a number of helper functions and that's fine, but they don't all want to be exposed here.

Yah these are good points... I guess I'll refactor.

Reply:
Simply return only the 1 or 2 functions that do the parsing, and attach them to the `worldeditadditions.parse` namespace. The others are cool, but they are internal and private to your parser. Other parts of the codebase do not need them, as they are specific to this use-case.

Reply:
I narrowed it down to 3.